docs/section22/Rule22-12.md

```diff
@@ -0,0 +1,47 @@
+
+# CHW&CW - Rule 22-12  
+
+**Schema Version:** 0.0.25    
+**Mandatory Rule:** True  
+**Rule ID:** 22-12  
+**Rule Description:** The heat rejection loop shall be modeled with a single cooling tower  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** G3.1.3.11
+**Data Lookup:** None  
+**Evaluation Context:** Each Heat Rejection Loop  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, 13, 7b, 8b, 12b.
+2. B-RMR is modeled with heat rejection loop
+
+**Function Call:**  
+
+1. get_heat_rejection_loops_connected_to_baseline_systems()
```

Comment:
@KarenWGard would this function be appropriate to use instead? https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/get_primary_secondary_loops_dict.md

Reply:
`get_primariy_seciondary_loops_dict` returns only loops that are correctly formatted as primary / secondary loops.  They way I've written the code, the user will get feedback about heat rejection equipment regardless of whether they modeled the primary / secondary loops correctly.  As a modeler, I prefer to resolve as many issues a possible in each iteration of modeling, so it was a personal choice to do it this way.

Reply:
I am not sure what you mean by "correctly formatted". I think the function returns all primary and all secondary loops associated with the baseline HVAC. So for example, if there is only primary but no secondary loop, that would be an incorrect arrangement but the RDS would still be verifying information based on the primary loop.

Reply:
@mkarpman - if a cooling loop has no secondary loop, then the function `get_primary_secondary_loops_dict` will not include a key for the primary loop.  So if I rely on this function for the rule, then the rule would return "not applicable"  

There is another rule that checks specifically for correct primary-secondary configuration, so I think we can give feedback here about the heat rejection system regardless of the chilled water loop configuration.